4-projects/README.md

```diff
@@ -9,6 +9,8 @@ The purpose of this step is to setup folder structure and projects for applicati
 1. 2-environments executed successfully.
 1. 3-networks executed successfully.
 1. Obtain the value for the access_context_manager_policy_id variable. Can be obtained by running `gcloud access-context-manager policies list --organization YOUR-ORGANIZATION_ID --format="value(name)"`.
+1. Obtain the values for the perimeter_name variable. Can be obteined by running
```

Comment:
`s/obteined/obtained/g`

We have also repeated this step 3 times `gcloud access-context-manager perimeters list`  - should we just add it once at the top?

Reply:
and the single version should be 
`gcloud access-context-manager perimeters list --policy ACCESS_CONTEXT_MANAGER_POLICY_ID --format="value(name)"`